src/scandeval/question_answering.py

```diff
@@ -420,7 +420,7 @@ def prepare_train_examples(
                 ):
                     token_start_index += 1
                 tokenized_examples.start_positions.append(token_start_index - 1)
-                while offsets[token_end_index][1] >= end_char:
+                while token_end_index > 0 and offsets[token_end_index][1] >= end_char:
```

Comment:
I am having a hard time judging whether this actually solves the problem - I assume that you already checked - but it might be good idea to write a test on such that we don't reintroduce the bug in the future?

Reply:
Wonderful exactly what I was looking for!